release-calendar/src/client/stylesheets/channelTable.scss

```diff
@@ -0,0 +1,118 @@
+.rtv-button {
+    float:left;
+    width: 35%;
+    height: 3em;
+    text-align: center;
+    padding-left: 0px;
+    padding-right: 0px;
+    background-color: white;
+}
+
+.label {
+    float: left;
+    width: inherit;
+    margin-left: 4em;
+    padding-right: 0px;
+    text-align: left;
+    position: absolute;
+    text-overflow: clip;
+    overflow: hidden;
+    white-space: nowrap;
+    line-height:3em;
+}
+
+.row {
+    float:left;
+    width:65%;
+    height: 3em;
+    box-sizing: border-box;
+    border-width: 1px;
+    border-style: solid;
+    border-color: rgb(216, 216, 216) rgb(209, 209, 209) rgb(186, 186, 186);
+  }
+
+@mixin square() {
+    display: none;
+	&:checked + i {
+		&:after {
+			opacity: .7;
+		}
+	}
+  + i {
+    box-sizing: border-box;
+    &:before, &:after {
+       box-sizing: border-box;
+    }
+  }
+	+ i {
+		position: relative;
+		display: inline-block;
+		width: 3em;
+	    height: 3em;
+		background: inherit;
+        
+		&:hover:after { opacity: .2; }
+
+		&:after {
+			content: '';
+			opacity: 0;
+			position: absolute;
+			top: .3em;
+			left: .3em;
+			width: 2.4em;
+			height: 2.4em;
+            background-color: white;
+            
+		}
+	}
+}
+
+label {
+  .square {
+    @include square();
+  }
+}
+
+//Colors: (to Esther: should I do label the colors this way or is it unnecessary)
```

Comment:
Yes, this is fine.

Side note - you can add comments to your own PR instead of writing notes in the code itself.